lib/addons/p5.dom.js

```diff
@@ -58,12 +58,23 @@
    * </code></div>
    *
    */
-  p5.prototype.select = function (e) {
+  p5.prototype.select = function (e, p) {
     var res;
     var str;
+    var container = document;
+    if (typeof p === 'string'){
+      if(p[0] !== '#'){
+        container = document.getElementById(p);
+      }else {
+        p = p.slice(1);
+        container = document.getElementById(p);
+      }
```

Comment:
I think we should treat it as an id. Since there can be more than one same type of tags and that would mean searching for element inside two parent elements making things complex.